shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitConstants.java

```diff
@@ -20,4 +20,9 @@ private ShortCircuitConstants() {
     public static final boolean DEFAULT_WITH_FEEDER_RESULT = true;
     public static final double DEFAULT_MIN_VOLTAGE_DROP_PROPORTIONAL_THRESHOLD = 0.0;
     public static final boolean DEFAULT_WITH_FORTESCUE_RESULT = true;
+    public static final double DEFAULT_SUB_TRANSIENT_COEFFICIENT = 70.0;
+    public static final boolean DEFAULT_WITH_LOADS = true;
+    public static final boolean DEFAULT_WITH_SHUNT_COMPENSATORS = true;
+    public static final boolean DEFAULT_WITH_VSC_CONVERTER_STATIONS = true;
+    public static final boolean DEFAULT_WITH_NEUTRAL_POSITION = true;
 }
```

Comment:
@JB-H, @annetill, do you have an opinion on what should be the default values for withLoads, withShuntCompensators, withVSCConverterStations and withNeutralPosition?

Reply:
As discussed: everything to true except withNeutralPosition